odoo/addons/base/models/qweb.py

```diff
@@ -311,7 +312,7 @@ def compile(self, template, options):
         try:
             # noinspection PyBroadException
             ns = {}
-            unsafe_eval(compile(astmod, '<template>', 'exec'), ns)
+            unsafe_eval(comprescompile(astmod, '<template>', 'exec'), ns)
```

Comment:
I don't think comprescompile is a proper python builtin
```suggestion
            unsafe_eval(compile(astmod, '<template>', 'exec'), ns)
```